### ISSUE TITLE
I have many warnings

### ISSUE THREAD

**Issue by Ruuuuddyyy** [drawRect/Instagram_Stories#82]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

<img width="276" alt="Screenshot 2020-01-15 at 16 38 19" src="https://user-images.githubusercontent.com/25419057/72438384-bf851680-37b5-11ea-9ace-1957b8dfa3dd.png">

How to fix this?

**Comment by ranmyfriend** [comment]
Hi @Ruuuuddyyy 

I appreciate your efforts on creating this issue. Sorry to say this. Currently, we are not supporting below 10.0. And if you look at our **ReadMe** file we mentioned iOS 10 and Xcode 8 is the base version to have this feature.

**Comment by Ruuuuddyyy** [comment]
Hi @ranmyfriend , but my project version iOS starts from 11.0 and xcode version 11.3.1

**Comment by ranmyfriend** [comment]
@Ruuuuddyyy 
Okay got it.
Currently, I have Xcode version 11.3. Let me update the version of 11.3.1 and will check and let you know.

**Comment by ranmyfriend** [comment]
Hi @Ruuuuddyyy 

Just now I have updated the Xcode Version and checked it works nicely from my side. 

Here is my combination: 
**Xcode Version 11.3.1 (11C504)
iOS 13.3 Simulator**

Can you please tell me more details about the bug. So I can help you

**Comment by Ruuuuddyyy** [comment]
I just drag and drop Source folder into my app and after that I have such a list of errors

**Comment by ranmyfriend** [comment]
Hi @Ruuuuddyyy 
Thanks for the suggestion. Just now I have created a sample project and copy-pasted the source folder got the warnings.

Related to warnings we have to support **iOS13** and regarding the layout error:
 You need this Enum which is located in **IGStoryPreviewController.swift**
```
public enum layoutType {
    case cubic
    var animator: LayoutAttributesAnimator {
        switch self {
        case .cubic:return CubeAttributesAnimator(perspective: -1/100, totalAngle: .pi/12)
        }
    }
}
```

Thanks for your support. We will fix the warnings and will let you know. give us some time

**Comment by ranmyfriend** [comment]
Hi @Ruuuuddyyy 
I have fixed the warnings related to iOS 13.0 and also **LayoutType** enum error also you won't get. Because I moved this enum into **Source** folder. Let me know if you face any issues.
